spec/controllers/scimitar/application_controller_spec.rb

```diff
@@ -208,7 +208,11 @@ def index
       context 'and bad JSON' do
         controller do
           def index
-            raise ActionDispatch::Http::Parameters::ParseError.new("Hello")
+            begin
+              raise 'Hello'
+            rescue
+              raise ActionDispatch::Http::Parameters::ParseError
```

Comment:
The Rails 7 constructor for this exception takes the message string as the parameter, but Rails 6 has no parameter on the constructor and instead initialises the message from `$!`. So, for Rails 6 compatibility, this change from the `v1` branch is retained - it works on Rails 7 also as the updated constructor's `message` parameter has a default of `$!.message`.